pages/data-providers/index.jsx

```diff
@@ -66,29 +70,40 @@ const getHelperMessage = ({ created, duplicated, error }) => {
 
   if (created) {
     return {
-      message: `We found OAI-PMH endpoint at ${created.oaiPmhEndpoint}`,
+      message: (
+        <>
+          We found {created.name} under the provided URL and saved it already.
+          We will start harvesting as soon as we are able to approve the data
+          provider.
+        </>
+      ),
       variant: 'success',
     }
   }
 
   if (duplicated) {
     return {
-      message:
-        `${duplicated.existingDataProviders[0].oaiPmhEndpoint} ${
-          duplicated.existingDataProviders.length > 1
-            ? `and ${duplicated.existingDataProviders.length - 1} more are`
-            : 'is'
-        } already` +
-        ' in our system. If you host more than one repository on the same domain,' +
-        ' please specify exact OAI-PMH endpoint',
+      message: (
+        <>
+          <a href={searchUrlFor(duplicated.existingDataProviders[0].id)}>
+            {duplicated.existingDataProviders[0].name}
+          </a>{' '}
+          {duplicated.existingDataProviders.length > 1
+            ? `and ${
+                duplicated.existingDataProviders.length - 1
+              } more are our data providers`
+            : 'is our data provider'}{' '}
+          already. If you host multiple repositories/journals on the same domain
+          please specify exact OAI-PMH endpoint or contact us at{' '}
+          <Link href={SUPPORT_EMAIL_URL}>{SUPPORT_EMAIL}</Link>.
+        </>
```

Comment:
... our data provider -> is in our data provider collection.

If you host multiple repositories or journals on the same domain please specify exact OAI-PMH endpoint or contact us at{' '}.

Reply:
Why do you propose 'collection'? It's expected result after clicking 'become a data provider' button. I feel it consistent.